hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/state/PersistenceManager.java

```diff
@@ -0,0 +1,46 @@
+package org.apache.mesos.hdfs.state;
+
+import org.apache.mesos.Protos;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Entry point for persistence for the HDFS scheduler.
+ */
+public interface PersistenceManager {
+
```

Comment:
Set and get frameworkId is already here which is a good thing. It should be removed from HdfsZkStore